base_location/views/res_city_view.xml

```diff
@@ -12,6 +12,7 @@
             </field>
             <field name="zipcode" position="after">
                 <field name="zip_ids" widget="many2many_tags" />
+                <field name="code" />
```

Comment:
For such field that are very optional, could you add optional option ?

Thanks !

Reply:
There are very few fields in the tree view, so I don't think it's a problem.